libs/@guardian/source-react-components-development-kitchen/src/file-input/FileInput.stories.tsx

```diff
@@ -64,6 +64,17 @@ const fileSizeValidation = (): ReactElement => {
 	);
 };
 
+const smallFileInput = (): ReactElement => {
```

Comment:
There seem to be three button sizes, IIRC: `default`, `small` and `xmall`. It is slightly confusing that this button is called `smallFileInput` but actually refers to an `xsmall` button. It would be good to have one of each, or make it a bit clearer?